internal/payloads/pubkey_payload.go

```diff
@@ -25,7 +26,8 @@ type PubkeyResponse struct {
 	FingerprintLegacy string `json:"fingerprint_legacy,omitempty" yaml:"fingerprint_legacy,omitempty"`
 }
 type PubkeyListResponse struct {
-	Data []*PubkeyResponse `json:"data" yaml:"data"`
+	Data     []*PubkeyResponse `json:"data" yaml:"data"`
+	Metadata *page.Metadata    `json:"metadata" yaml:"metadata"`
```

Comment:
I believe I did set a wrong standard of using pointers in our database model types, long term I would like to avoid pointers going forward. Little bit of copying is actually more performant than a pointer since the data can live on the stack and GC has less work. I’d say let’s keep it this way for now, this is an easy refactoring in the future.